libraries/rush-lib/src/cli/actions/InstallAction.ts

```diff
@@ -59,7 +59,12 @@ export class InstallAction extends BaseInstallAction {
       // it is safe to assume that the value is not null
       maxInstallAttempts: this._maxInstallAttempts.value!,
       // These are derived independently of the selection for command line brevity
-      filteredProjects: Array.from(await this._selectionParameters!.getSelectedProjectsAsync(this._terminal)),
+      filteredProjects: Array.from(
+        await this._selectionParameters!.getSelectedProjectsAsync(
+          this._terminal,
+          this._subspaceParameter?.value
```

Comment:
If subspace is a standard part of project selection, should we move it into `SelectionParameterSet`?

Reply:
We don't want to put it directly into SelectionParameterSet, as it would affect the build command as well, but I think what Pete said above makes sense to add it conditionally based on a constructor parameter